__tests__/app.test.js

```diff
@@ -133,6 +133,430 @@ describe('GET api/articles', () => {
     })
 });
 
+describe('GET api/articles?query', () => {
+    test('GET /api/articles?topic=cats - SINGLE VALID TOPIC QUERY: Status-200: returns object, key - articles, value - array filtered correctly by query given (1 result).', () => {
+        return request(app)
+            .get('/api/articles?topic=cats')
+            .expect(200)
+            .then(({body}) => {
+                const articles = body.articles
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)